src/ol-custom.json

```diff
@@ -28,21 +28,18 @@
       "externs/topojson.js"
     ],
     "define": [
-      "goog.array.ASSUME_NATIVE_FUNCTIONS=true",
       "goog.dom.ASSUME_STANDARDS_MODE=true",
-      "goog.json.USE_NATIVE_JSON=true",
+      "goog.DEBUG=false",
       "ol.ENABLE_DOM=false",
       "ol.ENABLE_WEBGL=false",
       "ol.ENABLE_PROJ4JS=false",
-      "ol.ENABLE_IMAGE=false",
-      "goog.DEBUG=false"
+      "ol.ENABLE_IMAGE=false"
     ],
     "jscomp_error": [
       "*"
     ],
     "jscomp_off": [
```

Comment:
I think v3.14.2 uses an older version of the compiler where `analyzerChecks` was not present.